app/controllers/bio_eddie_assets_controller.rb

```diff
@@ -38,6 +38,31 @@ def update
     end
   end
 
+  def license
+    license_file_path = Rails.root.join('data/bioeddie/license.cxl')
+    if File.file?(license_file_path)
+      send_file(license_file_path)
+    else
+      render_404
+    end
+  end
+
+  def bmt_request
+    return render_404 unless ENV['BIOMOLECULE_TOOLKIT_BASE_URL']
+
+    uri = URI.parse(ENV['BIOMOLECULE_TOOLKIT_BASE_URL'])
+    uri.path = request.original_fullpath.remove('/biomolecule_toolkit')
+
+    Net::HTTP.start(uri.host, uri.port, use_ssl: uri.scheme == 'https') do |http|
+      api_request = "Net::HTTP::#{request.request_method.capitalize}".constantize.new(uri)
+      api_request['Content-Type'] = 'application/json'
+      request_body = request.body.read
+      api_request.body = request_body if request_body.present?
+      api_response = http.request(api_request)
+      render body: api_response.body, content_type: api_response.content_type, status: api_response.code
+    end
+  end
```

Comment:
this probably doesn't belong in this branch, as it's present, and a bit different in another pull request

Reply:
This PR is based on the previous one https://github.com/biosistemika/scinote-web/pull/3414